### ISSUE TITLE
Installation issue on ubuntu 16.10

### ISSUE THREAD

**Issue by knspradeep** [snipe/snipe-it#3222]
#### Expected Behavior (or desired behavior if a feature request)

(what you expect to happen goes here)

Expected setup screen on browser but got "something went wrong error message"
#### Actual Behavior

(what actually happens goes here)

-----

#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode
- [ ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Please provide answers to these questions before posting your bug report:

- Version of Snipe-IT you're running=v3.6.2
- What OS and web server you're running Snipe-IT on = Ubuntu 16.10
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)=./install.sh
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is [reproduceable on the demo](https://snipeitapp.com/demo).
- Include any additional information you can find in `app/storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

 - Installing mtdowling/jmespath.php (2.3.0) Cloning 192f93e43c from cache
  - Installing psr/http-message (1.0.1) Cloning f6561bf28d from cache
  - Installing guzzlehttp/psr7 (1.3.1) Cloning 5c6447c9df from cache
  - Installing guzzlehttp/promises (1.2.0) Cloning c10d860e2a from cache
  - Installing guzzlehttp/guzzle (6.2.1) Cloning 3f808fba62 from cache
  - Installing aws/aws-sdk-php (3.19.2) Cloning 3cb9041312 from cache
  - Installing aws/aws-sdk-php-laravel (3.1.0) Cloning 3b946892d4 from cache
  - Installing maximebf/debugbar (v1.12.0) Cloning e634fbd32c from cache
  - Installing barryvdh/laravel-debugbar (V2.2.3) Cloning ecd1ce5c4a from cache
  - Installing doctrine/lexer (v1.0.1) Cloning 83893c552f from cache
  - Installing doctrine/annotations (v1.2.7) Cloning f25c8aab83 from cache
  - Installing doctrine/cache (v1.5.4) Cloning 47cdc76ceb from cache
  - Installing doctrine/collections (v1.3.0) Cloning 6c1e4eef75 from cache
  - Installing doctrine/common (v2.5.3) Cloning 10f1f19651 from cache
  - Installing doctrine/dbal (v2.4.2) Cloning fec965d330 from cache
  - Installing erusev/parsedown (dev-master f671ae7) Cloning f671ae7364 from cache
  - Installing fideloper/proxy (3.1.0) Cloning ec4dd30141 from cache
  - Installing intervention/image (dev-master 45a41a3) Cloning 45a41a38bd from cache
  - Installing laravelcollective/html (v5.2.4) Cloning 3a312d39ff from cache
  - Installing league/csv (7.2.0) Cloning 69bafa6ff9 from cache
  - Installing maknz/slack (dev-master e9bab77) Cloning e9bab77eb7 from cache
  - Installing misterphilip/maintenance-mode (1.0.2) Cloning 35e061703e from cache
  - Installing neitanod/forceutf8 (dev-master 2c1b21e) Cloning 2c1b21e00e from cache
  - Installing christian-riesen/base32 (1.3.1) Cloning 0a31e50c0f from cache
  - Installing pragmarx/google2fa (v1.0.1) Cloning b346dc1383 from cache
  - Installing spatie/db-dumper (1.5.1) Cloning efdf41891a from cache
  - Installing spatie/laravel-backup (3.8.1) Cloning 08b7b8c778 from cache
  - Installing tecnickcom/tc-lib-color (1.12.0) Cloning 176464ae7a from cache
  - Installing tecnickcom/tc-lib-barcode (dev-master df69541) Cloning df69541618 from cache
  - Installing watson/validating (2.2.2) Cloning 8f37e416aa from cache
Generating autoload files
> php artisan clear-compiled
> php artisan optimize
Generating optimized class loader
Compiling common classes

  ***If you want mail capabilities, edit /var/www/snipeit/.env and edit based on .env.example***

  ***Open http://192.168.29.236 to login to Snipe-IT.***

* Cleaning up...
* Finished!

**Comment by snipe** [comment]
Please show us the contents of your error log, per the getting help documentation. https://snipe-it.readme.io/docs/getting-help

**Comment by taylordrake** [comment]
Also, you should enable de-bug mode in your .env file by changing the value from `false` to '`true`

The navigate back to your "Something Went Wrong" page, it should have an error message available there that should provide more information.

**Comment by knspradeep** [comment]
Hey sorry for the delay after changing the /etc/apache2/sitesavailable/000-default.conf

Where the line "DocumentRoot" is, change the location to "/var/www/snipeit/public". Write the changes to the file and exit.

**Comment by pbulteel** [comment]
Hi,

I had a few issue at first as well, but I managed to get through it. These are the issues I encountered since I was doing a manual install.

1. Using PHP7, so had to manually select all the PHP7 modules needed. 

php7.0-cli					install
php7.0-common					install
php7.0-curl					install
php7.0-gd					install
php7.0-json					install
php7.0-mbstring					install
php7.0-mcrypt					install
php7.0-mysql					install
php7.0-opcache					install
php7.0-readline					install
php7.0-xml					install
php7.0-zip					install

2. Wrote a /etc/apache2/sites-available/snipe-it.conf and then did "sudo a2ensite snipe-it"

3. Used "certbot" to get an SSL certificate (using LetsEncrypt.org - free SSL) 

4. Enabled "debug=true" in the .env so that I could troubleshoot - this pointed to 5. 

5. Mysql (or mariadb in my case) was using the socket plugin for the user. I had to log into mariadb as root and then do the grant on the user for snipeit again. Check the status by first doing a "select host, user, password, plugin from user;" If you see that plugin has something, likely password will be blank. 

--
Patrick
